serdes-native/.settings/org.eclipse.buildship.core.prefs

```diff
@@ -0,0 +1,13 @@
+arguments=
```

Comment:
Unnecessary file. Seems like these files are generated from eclipse. Shall we look into these and remove all the unnecessary files?